lua/lspconfig/lelwel_ls.lua

```diff
@@ -0,0 +1,25 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.lelwel_ls = {
+  default_config = {
+    cmd = { 'lelwel-ls' },
+    filetypes = { 'llw' },
+    root_dir = util.path.dirname,
+  },
+  docs = {
+    description = [[
+https://github.com/0x2a-42/lelwel
+
+Language server for lelwel grammars.
+
+You can install `lelwel-ls` via cargo:
+```sh
+cargo install --features="lsp" lelwel
+```
+]],
+    default_config = {
+      root_dir = "vim's starting directory",
```

Comment:
The default_config is not accurate, you can just leave it off.

Reply:
OK, I removed it.